uktrade/data-hub-frontend
# Show registered address in search results if it is different to main address

## Description of change

When you search for a company, you will now see the registered address in the search results where that is different to the main address.

## Test instructions

Use search to find a company that has a different registered address to main address - you should see both addresses in the search address. If the two addresses are the same, you should only see the main address. If there is no registered address, you should only see the main address.

## Screenshots

![image](https://user-images.githubusercontent.com/1234577/140765354-2015422c-4bed-43ed-afcd-ef255b10c7da.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
